app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -24,9 +24,31 @@ angular.module('miq.util').factory('chartsMixin', [function chartMixinFactory ()
     }
   };
 
+  var processHeatmapData = function(data) {
+    data.sort(function(a, b) {return b.value - a.value})
+    var result = [];
+
+    for (var i = 0, l = data.length; i < l; i++) {
+      var d = data[i]
+      var percent = d.value * 100
+      var used = Math.floor(d.value * d.info.total)
+      var available = d.info.total - used
+      var tooltip =`${d.info.node} : ${d.info.provider}\n${percent}%: ${used} Used of ${d.info.total} Total\n - ${available} Available`
```

Comment:
> he node name takes the entire line, but the provider name along with percentage takes the next line.

Im guessing that theres a max width for the tooltip so the provider got bunked down @jeff-phillips-18



> the colon near percent has no space to the left of it, while it does have spaces from both sides near d.info.node.

Copied from mock up. I think in the case of first colon its more like a delimiter between the node and its provider while the second is more like "here are the details of this percent" .
 

> I wouldn't capitalize 'Used' as this is the middle of the sentence here.

Ill change it.